xbmc/guilib/GUIToggleButtonControl.cpp

```diff
@@ -38,6 +39,7 @@ void CGUIToggleButtonControl::Process(unsigned int currentTime, CDirtyRegionList
     m_selectButton.SetPulseOnSelect(m_pulseOnSelect);
     ProcessToggle(currentTime);
     m_selectButton.DoProcess(currentTime, dirtyregions);
+    CGUIControl::Process(currentTime, dirtyregions);
```

Comment:
no clue about `guilib` :) but is `CGUIControl::Process()` the correct call?
i have tested this one

```suggestion
    CGUIButtonControl::Process(currentTime, dirtyregions);
```

which also seems to fix the issue at a first glance.
it would also allow to simply remove the `else` then??